acceptance-tests/aurora_mysql_data_migration_test.go

```diff
@@ -2,7 +2,7 @@ package acceptance_tests_test
 
 import (
 	"csbbrokerpakaws/acceptance-tests/helpers/apps"
-	"csbbrokerpakaws/acceptance-tests/helpers/dms"
+	"csbbrokerpakaws/acceptance-tests/helpers/awscli/dms"
```

Comment:
I would have been inclined to leave the `dms` package as it was and just factor out one function into `awscli`, but I don't see any harm in moving it.